Content.Shared/Mech/SharedMech.cs

```diff
@@ -60,3 +60,10 @@ public sealed partial class MechOpenUiEvent : InstantActionEvent
 public sealed partial class MechEjectPilotEvent : InstantActionEvent
 {
 }
+
+[Serializable, NetSerializable]
+public sealed class MechEnergyMessage(float charge, float maxEnergy) : BoundUserInterfaceMessage
+{
+    public float Charge { get; } = charge;
+    public float MaxEnergy { get; } = maxEnergy;
+}
```

Comment:
Why not
```suggestion
public sealed class MechEnergyMessage(float charge, float maxEnergy) : BoundUserInterfaceMessage
{
    public readonly float Charge = charge;
    public readonly float MaxEnergy = maxEnergy;
}
```
?
This is less of a suggestion and more of a "I don't know what's better, so I'd appreciate insight" question.

Reply:
Will cause CS0106.

Reply:
I believe that CS0106 would be raised if it were
```csharp
readonly float Charge { get; } = charge;
```
, because that's a property with a synthetic getter _method_.
I'm suggesting a normal field that's readonly:
```csharp
readonly float Charge = charge;
```

Did you not remove the `{ get; }`, or am I misunderstanding?

Reply:
Done.